src/main/java/duke/ItemList.java

```diff
@@ -0,0 +1,10 @@
+package duke;
+
+import duke.items.Item;
+import java.util.LinkedList;

```

Comment:
Wrong import statement order.
```suggestion
import java.util.LinkedList;

import duke.items.Item;
```